docs/_data/mesheryctlcommands/meshes/deploy/command/deploy.yaml

```diff
@@ -0,0 +1,5 @@
+name: deploy
+description: deploy a service mesh into the cluster
```

Comment:
Maybe we can make the capitalize the first letter of the description. @alphaX86 what's your view on this and if we can maintain the consistency in all over the docs.

Reply:
@sayantan1413 it'd require a PR from anyone to do this... Either we can resolve this by ourselves or make an issue for Newcomers to engage with that